openprescribing/frontend/management/commands/send_monthly_alerts.py

```diff
@@ -69,7 +69,11 @@ def get_org_bookmarks(self, now_month, **options):
         """
         query = (
             Q(approved=True, user__is_active=True) &
-            ~Q(user__emailmessage__tags__contains=['measures', now_month]))
+            ~Q(user__emailmessage__tags__contains=['measures', now_month]) &
+            # Only include bookmarks for either a practice or pct: when both
+            # are NULL this indicates an All England bookmark
```

Comment:
Why do we want to send them separately?

Reply:
I think it's certainly helpful while we're still working on these to be able to send our standard alerts without sending the national ones. But even longer term it feels "cleaner" (in some unspecified sense) to keep the code sending the different alert types separate.